src/components/CharacterBackground.js

```diff
@@ -1,35 +1,36 @@
-import React, { Component } from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
 import DungeonService from '../services/dungeonService';
 
-export default class CharacterBackground extends Component {
-  constructor() {
-    super();
-    // Hard Coded for quick development
-    const characterId = '6oHp62hgG0zeFPjwa8RB';
-    this.state = { characterId };
-  }
-
-  componentWillMount() {
-    this.getCharacterInfo(this.state.characterId);
-  }
+function CharacterBackground() {
+  // Hard Coded for quick development
+  // Also linting disabled until we add a way to grab the ID
+  // eslint-disable-next-line
+  const [characterId, updateCharacterId] = useState('6oHp62hgG0zeFPjwa8RB');
```

Comment:
How do you imagine the ID is set? Through state? Or url param? I could see it either way